lib/perl/Genome/Model/Build/ReferenceSequence/Converter.pm

```diff
@@ -40,6 +40,10 @@ class Genome::Model::Build::ReferenceSequence::Converter {
             is_optional => 1,
         },
     ],
+    has_transient_optional => [
+        _chr_map_hash_ref => {
+        },
```

Comment:
It'd be nice if this property included:
```perl
is => 'HASH'
```
(The type needn't necessarily be included in the name then, but either way!)

A `doc` line might be good, too, but is less important for internal things.